packages/common-types/src/baseTypes/commonTypes.ts

```diff
@@ -125,3 +125,7 @@ export type FromJSONConstructor = {
 };
 
 export type DeviceShareDescription = { module: string; userAgent: string; dateAdded: number; customDeviceInfo?: string };
+
+// This has to be fixed to secp256k1 for now because it is being used in
+// combination with eccrypto's `encrypt` and `decrypt`, which is secp256k1 only.
+export const FACTOR_KEY_TYPE = KeyType.secp256k1;
```

Comment:
If we use just one key type for tkey and tss, then we should also adapt the same key type for the factor keys?

Reply:
i think factorKeys should always be "secp256k1" irrespective of keyType , so it would be better to keep it seperate imo